src/js/pre-need/containers/ConfirmationPage.jsx

```diff
@@ -14,29 +15,83 @@ const scrollToTop = () => {
 };
 
 class ConfirmationPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { isExpanded: false };
+  }
+
   componentDidMount() {
     focusElement('.confirmation-page-title');
     scrollToTop();
   }
 
+  toggleExpanded = (e) => {
+    e.preventDefault();
+    this.setState({ isExpanded: !this.state.isExpanded });
+  }
+
   render() {
+    const form = this.props.form;
+    const response = this.props.form.submission.response
+      ? this.props.form.submission.response.attributes
+      : {};
+    const name = form.data.application.claimant.name;
+    const submittedAt = moment(form.submission.timestamp);
+    const offset = submittedAt.isDST() ? '-0500' : '-0600';
```

Comment:
This is going to show the date in CT, rather than the user's timezone. I think we may want to change the way the date is handled to how it is on the edu confirmation pages.

Reply:
Thanks @jbalboni, removed. Should I do the same for the burial confirmation page?